charset_normalizer/md.py

```diff
@@ -153,9 +137,8 @@ def eligible(self, character: str) -> bool:
 
     def feed(self, character: str) -> None:
         if (
-            character not in {"\n", "\t", "\r", "\v"}
+            character.isspace() is False  # includes \n \t \r \v
```

Comment:
Why `... is False` instead of `not`? 

```
$ python -m timeit '"x".isspace() is False'
10000000 loops, best of 5: 39.6 nsec per loop
$ python -m timeit 'not "x".isspace()'
10000000 loops, best of 5: 37.2 nsec per loop
```

Reply:
It doesn't really make a difference, does it?

Reply:
Let's leave it for @Ousret to decide

Reply:
Lets keep is False. The diff is too small to consider any changes here.